.github/workflows/build.yml

```diff
@@ -125,6 +125,7 @@ jobs:
             echo "Smoke Test Failed"
             exit 1
           fi
+          sleep 10
```

Comment:
There is a bug where the ZetaClient waits indefinitely without using this sleep. I can provide more log. There might be an eventual issue to fix that I think could be fixed in another PR

Reply:
OK sweet. Yea an indefinite wait is far more pricey than 10 seconds =)

Reply:
I removed the 10 seconds sleep, and somehow the issue is no longer here. It may have been related to Github or somehow I introduced a fix at some point.

Reply:
Added back the 10 seconds as it was failing again... Created an issue for this.